packages/apputils/src/iframe.ts

```diff
@@ -24,6 +48,42 @@ export class IFrame extends Widget {
   set url(url: string) {
     this.node.querySelector('iframe')!.setAttribute('src', url);
   }
+
+  private _sandbox: IFrame.SandboxExceptions[] = [];
+}
+
+/**
+ * A namespace for IFrame widget statics.
+ */
+export namespace IFrame {
+  /**
+   * Exceptions to the iframe sandboxing policies.
+   * These are specified here:
+   * https://www.w3schools.com/tags/att_iframe_sandbox.asp
```

Comment:
Probably cite a more authoritative source (whatwg or W3C):
- https://www.w3.org/TR/2011/WD-html5-20110525/the-iframe-element.html#attr-iframe-sandbox
- https://html.spec.whatwg.org/multipage/iframe-embed-object.html#attr-iframe-sandbox

And offer a more reputable source the user (MDN).
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox

Some day IN THE FUTURE.